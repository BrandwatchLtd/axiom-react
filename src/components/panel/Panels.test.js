import React from 'react';
import renderer from 'react-test-renderer';
import { Panels, Panel } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <Panels { ...props }>
      <Panel>Lorem Ipsum</Panel>
      <Panel>Lorem Ipsum</Panel>
    </Panels>
  );
}

describe('Panels', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
