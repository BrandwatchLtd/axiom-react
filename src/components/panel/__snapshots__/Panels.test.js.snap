// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panels renders with defaultProps 1`] = `
<div
  className="ax-panels ax-theme--light"
>
  <div
    className="ax-panel ax-theme--light"
    style={
      Object {
        "minHeight": undefined,
      }
    }
  >
    Lorem Ipsum
  </div>
  <div
    className="ax-panel ax-theme--light"
    style={
      Object {
        "minHeight": undefined,
      }
    }
  >
    Lorem Ipsum
  </div>
</div>
`;
